dcgm/README.md

```diff
@@ -0,0 +1,260 @@
+# Agent Check: DCGM
+
+## Overview
+
+Harnessing the Nvidia DCGM Exporter, this check monitors the exposed through the Datadog Agent.
```

Comment:
Change to `this check monitors the exposed **metrics** through the Datadog Agent.`

Reply:
nice catch! I've rewritten that whole sentence to address it, lmk what you think.